src/common/utils.c

```diff
@@ -281,6 +281,13 @@ int __Extrae_Utils_rename_or_copy (char *origen, char *desti)
 			/* Remove the files */
 			unlink (origen);
 		}
+#if defined (OS_RTEMS)
```

Comment:
This modifies the behaviour when the OS is RTEMS to force removing the file before renaming.

Reply:
As far as I remember using the Gr740 with NFS caused errors if you try to rename a file to other name that already exists, so what we do is to check before renaming if there is any file with the destination name and delete it, this should not change the behavior of the function.